modules/installation-overview.adoc

```diff
@@ -77,7 +77,7 @@ of new certificates or keys, after installation.
 == Available platforms
 
 In {product-title} {product-version}, you can install a cluster that uses
-installer-provisioned infrastructure on only Amazon Web Services (AWS).
+installer-provisioned infrastructure on Amazon Web Services (AWS) and OpenStack.
```

Comment:
Please defer to the fix for this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1756513

Reply:
I'll revert this.